docs/usage/administration/admin_api/README.md

```diff
@@ -5,9 +5,18 @@
 Many of the API calls in the admin api will require an `access_token` for a
 server admin. (Note that a server admin is distinct from a room admin.)
 
-A user can be marked as a server admin by updating the database directly, e.g.:
+An existing user can be marked as a server admin by updating the database directly.
 
+When using PostgreSQL, switch to the `postgres` user and start the command line tool.
+```sh
+su - postgres
+psql
+```
+
+Check your  [database settings](config_documentation.md#database) in the `homeserver.yaml`, connect to the
```

Comment:
I'm unconvinced this sentence is needed, if you are running a Synapse instance but don't know where to find the name of database you've configured I'd say you're doing something wrong. But I guess more documentation doesn't hurt.

I would however like to see "the `homeserver.yaml`" replaced with e.g. "configuration file" since not everyone is going to name their configuration file `homeserver.yaml`.